specification/databricks/resource-manager/Microsoft.Databricks/preview/2023-09-15-preview/databricks.json

```diff
@@ -0,0 +1,2025 @@
+      "description": "Schedule for automatic restart of compute resources",
+      "properties": {
+        "weekDayFrequency": {
+          "type": "string",
+          "description": "Frequency of execution of cluster updates",
+          "enum": [
+            "None",
+            "FirstOfMonth",
+            "SecondOfMonth",
+            "ThirdOfMonth",
+            "FourthOfMonth",
+            "FirstAndThirdOfMonth",
+            "SecondAndFourthOfMonth"
+          ],
+          "x-ms-enum": {
+            "modelAsString": true,
+            "name": "WeekDayFrequency"
+          }
+        },
+        "weekDay": {
+          "type": "string",
+          "description": "Day of the week for automatic cluster updates to occur",
+          "enum": [
+            "None",
+            "Monday",
+            "Tuesday",
+            "Wednesday",
+            "Thursday",
+            "Friday",
+            "Saturday",
+            "Sunday"
+          ],
+          "x-ms-enum": {
+            "modelAsString": true,
+            "name": "WeekDay"
+          }
+        },
+        "startTimeWindow": {
+          "type": "object",
+          "description": "Configuration of time of day when automatic cluster updates should occur",
+          "$ref": "#/definitions/StartTimeWindowDefinition"
+        }
+      }
+    },
+    "StartTimeWindowDefinition": {
+      "type": "object",
+      "description": "Configuration of time of day when automatic cluster updates should occur",
+      "properties": {
+        "hours": {
+          "format": "int32",
+          "type": "integer",
+          "description": "Hour of the day when automatic cluster updates should occur"
+        },
+        "minutes": {
+          "format": "int32",
+          "type": "integer",
+          "description": "The minute when automatic cluster updates should occur"
+        }
+      }
+    },
+    "ComplianceSecurityProfileDefinition": {
+      "type": "object",
+      "description": "Feature that helps customers meet the applicable security requirements of some compliance standards.",
+      "properties": {
+        "complianceStandards": {
+          "description": "Compliance standards associated with the workspace.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ComplianceStandardDefinition"
+          }
+        },
+        "value": {
+          "type": "string",
+          "description": "Status of Compliance Security Profile feature",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "modelAsString": true,
+            "name": "ComplianceSecurityProfileValue"
+          }
+        }
+      }
+    },
+    "ComplianceStandardDefinition": {
+      "description": "Compliance standard that can be associated with a workspace.",
+      "type": "string",
+      "enum": [
+        "HIPAA",
+        "PCI_DSS",
+        "NONE"
+      ],
+      "x-ms-enum": {
```

Comment:
>x-ms-enum [](http://example.com/codeflow?start=7&length=9)

Consider adding descriptions for each value. Ref: [https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-enum](https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-enum).